cattqt/cattqt.py

```diff
@@ -697,7 +698,6 @@ def play(self, d, text):
             print('Failed to play "%s" please include full path' % text)
             return
         self.on_stop_signal(d)
-        d.stopping_timer.stop()
```

Comment:
this and

Reply:
> > I do have a question though. The last commit on this repository was on March 10, 2020 and version 3.0 was released on March 15, 2020. Why does it not include the commit from March 10, because I noticed that my commit seems to undo the one line change from March 10?
> 
> There are the commits here but not sure how they are not on a branch and not on master, huh.. https://github.com/soreau/catt-qt/commits/v3.0

Thanks, but yeah that is weird. In the link you provided. I saw a commit for March 15th, but when I'm on the master branch which you land on by default when you click the url to this repo. If I then click on commits. The last commit is from March 10, that is kinda confusing. I do see the commit from the 15th in the actual file though from cloning the repo. I guess I just don't know how github works.